scheduler.go

```diff
@@ -280,10 +279,17 @@ func (j *Job) GetAt() string {
 	return fmt.Sprintf("%d:%d", j.atTime/time.Hour, (j.atTime%time.Hour)/time.Minute)
 }
 
-// From schedules the next run of the job
-func (s *Scheduler) From(t *time.Time) *Scheduler {
-	currentJob := s.getCurrentJob()
-	currentJob.nextRun = *t
+// StartAt schedules the next run of the job
+func (s *Scheduler) StartAt(t time.Time) *Scheduler {
+	s.getCurrentJob().nextRun = t
+	return s
+}
+
+// NextTick returns a pointer to a time that will run at the next tick
+func (s *Scheduler) StartImmediately() *Scheduler {
+	job := s.getCurrentJob()
+	job.nextRun = time.Now().In(s.loc)
```

Comment:
Do we need to add a small buffer here in case between defining the jobs and starting the scheduler there was some small latency?

Reply:
I don't know about that... any assumption between the time of the job's definition and scheduler's start here would be incorrect since it could be of any value. By setting nextRun to now, it assumes it needs to run as soon as the next scheduler's tick occurs

Reply:
👍